aemcmc/nuts.py

```diff
@@ -0,0 +1,129 @@
+import aesara
+from aehmc import nuts as aehmc_nuts
+from aehmc.utils import RaveledParamsMap
+from aeppl import joint_logprob
+from aeppl.transforms import RVTransform, TransformValuesOpt, _default_transformed_rv
+from aesara.tensor.random import RandomStream
+from aesara.tensor.var import TensorVariable
+
+from aemcmc.utils import ModelInfo
+
+
+def nuts(
+    srng: RandomStream,
+    model: ModelInfo,
+    inverse_mass_matrix: TensorVariable,
+    step_size: TensorVariable,
+):
```

Comment:
An explicit return type would be helpful here (primarily for users and developers).